code/site/libraries/plugin.php

```diff
@@ -476,7 +476,7 @@ final private function log()
 	 *
 	 * @since 1.0
 	 */
-	final private function lastUsed()
+	private function lastUsed()
```

Comment:
@xRealNeon functions are deliberately made final here. 
can you please specify the case in which you need to remove final keyword?

Reply:
Private methods cannot be final as they are never overridden by other classes